test/Microsoft.Extensions.Internal.Test/Microsoft.Extensions.Internal.Test.csproj

```diff
@@ -17,15 +18,19 @@
     <Compile Include="..\..\shared\Microsoft.Extensions.PropertyActivator.Sources\**\*.cs" />
     <Compile Include="..\..\shared\Microsoft.Extensions.PropertyHelper.Sources\**\*.cs" />
     <Compile Include="..\..\shared\Microsoft.Extensions.SecurityHelper.Sources\**\*.cs" />
+    <Compile Include="..\..\shared\Microsoft.Extensions.StackTrace.Sources\StackFrame\**\*.cs" />
     <Compile Include="..\..\shared\Microsoft.Extensions.TypeNameHelper.Sources\**\*.cs" />
     <Compile Include="..\..\shared\Microsoft.Extensions.WebEncoders.Sources\**\*.cs" />
     <EmbeddedResource Include="..\..\shared\*.Sources\**\*.resx" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\ThrowingLibrary\ThrowingLibrary.csproj" />
```

Comment:
Regardless of what kind of pdb you generate, exceptions seem preserve stack trace as long as they are produced and caught in the same assembly. I had to cause it to be produced in a different assembly for us to actually use the portable pdb reader. I'll make a note of this in the test.